frontend/package.json

```diff
@@ -27,7 +27,7 @@
   },
   "dependencies": {
     "@openshift/dynamic-plugin-sdk": "^1.0.0-alpha16",
-    "@openshift/dynamic-plugin-sdk-utils": "^1.0.0-alpha19",
+    "@openshift/dynamic-plugin-sdk-utils": "^1.0.0",
```

Comment:
@vojtechszocs FYI this will cause minor/patch version publishes to automatically be picked up by HAC. In the future any breaking changes will need to be a major version - or we need to scope down our range here.